clients/admin-ui/src/features/messaging-templates/messaging-templates.slice.ts

```diff
@@ -15,13 +19,68 @@ export type BulkPutMessagingTemplateResponse = {
   failed: BulkUpdateFailed[];
 };
 
+export type MessagingTemplateResponse = {
+  type: string;
+  id?: string; // not on default response
+  is_enabled: boolean;
+  content: {
+    subject: string;
+    body: string;
+  };
+  properties?: MinimalProperty[]; // not on default response
+};
+
+export type MessagingTemplateDefaultResponse = {
+  type: string;
+  is_enabled: boolean;
+  content: {
+    subject: string;
+    body: string;
+  };
+};
```

Comment:
Feels like some of these type declarations can be simplified into interfaces that extend the MessagingTemplateDefault type (that's generated from the API) to avoid redundancy.